index.html

```diff
@@ -5,14 +5,81 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>My Quiz App</title>
     <link rel="stylesheet" href="styles/index.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link href="/fonts/css/all.css" rel="stylesheet" />
   </head>
   <body class="app-grid">
-    <header class="app-grid__header">
-
-      <h1 class="">My Quiz App</h1>
-      
+    <header class="header">
+      <h1 class="header__heading">My Quiz App</h1>
```

Comment:
Very good that you renamed the class from `app-grid__header` to `header`. Here I would suggest `header__headline` instead of `header__heading`.
```suggestion
      <h1 class="header__headline">My Quiz App</h1>
```